hazelcast-sql-core/src/main/java/com/hazelcast/sql/impl/calcite/opt/physical/visitor/PlanCreateVisitor.java

```diff
@@ -318,6 +332,49 @@ public void onFilter(FilterPhysicalRel rel) {
         pushUpstream(filterNode);
     }
 
+    @Override
+    public void onSortMergeExchange(SortMergeExchangePhysicalRel rel) {
+        PlanNode upstreamNode = pollSingleUpstream();
+        PlanNodeSchema upstreamNodeSchema = upstreamNode.getSchema();
+
+        // Create sender and push it as a fragment.
+        int edge = nextEdge();
+
+        int id = pollId(rel);
+
+        UnicastSendPlanNode sendNode = new UnicastSendPlanNode(
+            id,
+            upstreamNode,
+            edge,
+            AllFieldsRowPartitioner.INSTANCE
```

Comment:
Since our current implementation always pass values to a single node (ROOT), we do not need `AllFieldsRowPartitioner.INSTANCE`, because there always be only one target member, and any partition will be assigned to that member. We may introduce a static partitioner, that assigns zero partition to every row.

Reply:
Fixed in https://github.com/hazelcast/hazelcast/pull/18013/commits/ba2cf9f28770bf86017f73cd451ee28e07125f19